src/layer.cpp

```diff
@@ -199,7 +199,16 @@ Clef *Layer::GetClef(LayerElement *test)
         assert(clef);
         return clef;
     }
-
+    Doc *doc = dynamic_cast<Doc *>(this->GetFirstParent(DOC));
```

Comment:
Can you make this a distinct method `Layer::GetClefFacs()` and call it from here when necessary?